.github/workflows/python-release.yml

```diff
@@ -9,18 +9,21 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
         python-version: '3.x'
-    - name: Install pypa/build
+
+    - name: Install poetry
       run: |
-        python -m pip install build --user
+        python -m pip install poetry
+
     - name: Build a binary wheel and a source tarball
       run: |
-        python -m build --sdist --wheel --outdir dist/ .
+        python -m poetry build
+
     - name: Publish to PyPI
-      uses: pypa/gh-action-pypi-publish@master
+      uses: actions/pypi-publish@release/v1
```

Comment:
https://github.com/pypa/gh-action-pypi-publish documentation says one should use `pypa/gh-action-pypi-publish`, is `actions/pypi-publish@release/v1` an alias for that?

Sorry if this question is stupid, I don't use github actions often

Reply:
`actions/pypi-publish` is an alias for `pypa/gh-action-pypi-publish`. I just used the alias because it looks more official.
We might also consider using pinned commit hashes and dependabot